tests/integration/helpers.bash

```diff
@@ -18,7 +18,7 @@ HELLO_IMAGE="$TESTDATA/hello-world.tar"
 HELLO_BUNDLE="$BATS_TMPDIR/hello-world"
 
 # CRIU PATH
-CRIU="$(which criu)"
+CRIU="$(which criu)" || true
```

Comment:
> (CRIU="$(which criu)") || true

Yeah, I don't really like it. Can you instead use

```
CRIU="$(which criu || true)"
```

I also don't really like that `$CRIU` is now set to `criu not found` rather than an empty string, but what are you going to do...